tools/ci/install-r-dependencies.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Context: .github/workflows/*.yaml
+export R_REMOTES_NO_ERRORS_FROM_WARNINGS='true'
+Rscript -e 'install.packages(c("remotes", "rcmdcheck"))'
```

Comment:
You don't actually _use_ the `rcmdcheck` package as you invoke `R CMD check --as-cran --no-manual --ignore-vignettes tiledb_*.tar.gz`.  So you don't need to install it.